app/controllers/conversations_controller.rb

```diff
@@ -24,7 +24,7 @@ def index
     gon.contacts = contacts_data
 
     respond_with do |format|
-      format.html
+      format.html { render "index", locals: {no_contacts: current_user.contacts.empty?} }
```

Comment:
I think we should check if the mutual contacts are empty since you are only allowed to start a conversation with mutual contacts.